docs/docs/user-guide/cloud-storage.mdx

```diff
@@ -62,6 +62,13 @@ Rasa supports using [S3](https://aws.amazon.com/s3/) ,
 
   If there is no container with the name `AZURE_CONTAINER`, Rasa will create it.
 
+* Other Remote Storage
+
+  Other remote storage is supported using the external model persistor library,
+  which you can install or implement by yourself.
+
+  Start the Rasa server with ``remote-storage`` option set to the module path of model persistor.
```

Comment:
I suggest to give an example of how the command would look like

Reply:
I had add a single command